test/run_test.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 
-set -u
-set -e
+#set -u
+#set -e
```

Comment:
I think this change will make CI pass even after failure on tests.

Reply:
Thats true. The problem i was trying to address is that script should continue to upload aperturedb stuff to s3 even after failures in prev steps . I suppose I need to do a bash OR or something

Do you have a better idea?

Reply:
nice, this reminded me I had the same issues. Here is how I solved it: https://github.com/aperture-data/athena/blob/develop/benchmarks/docker/scripts/start.sh#L49 (also, check the comment at the top of the file 🤣 )